CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,2 @@
+[cygnus-ngsi][ArcgisFeatureTableSink]New sink to persist Arcgis data. (#1672)
+[cygnus-arcgis][new backend]New Rest based backend to persist Arcgis data. (#1672)
```

Comment:
```suggestion
[cygnus-ngsi][ArcGis] New sink (rework) to persist Arcgis data (#1672)
[cygnus-common][ArcGis] New Rest based backend to persist Arcgis data. (#1672)
```

Not sure, but I think is closer to the "changelog style", as seen in https://github.com/telefonicaid/fiware-cygnus/releases?after=FIWARE_7.8.2

Not sure about cygnus-commons, but I'd say that backends are included in tha part, isn't it?